packages/terra-application-header-layout/package.json

```diff
@@ -29,17 +29,19 @@
     "terra-base": "^2.5.0"
   },
   "devDependencies": {
-    "terra-props-table": "^1.10.1"
+    "terra-props-table": "^1.10.1",
```

Comment:
With the new site, we can drop all props-table dependencies & scripts from packages.